CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,8 @@
+We have picked the following conduct guideline based on [The Ruby Community Conduct Guideline](https://www.ruby-lang.org/en/conduct/).
+
+This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to the Ruby community. It applies to all “collaborative space”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
```

Comment:
```suggestion
This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to the community. It applies to all “collaborative space”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
```

I think this is ok to omit.
First thought was to put "flake-parts" there, but I'd like to de-emphasize that. `flake-parts` itself is intentionally very minimal and almost no collaboration happens there, so I'd rather refer to "the community" because they're doing far more useful things than `flake-parts` does.